src/Angor/Client/Pages/Invest.razor

```diff
@@ -472,6 +472,9 @@
 
     private async Task HandleSignatureReceivedAsync(string? nostrPrivateKeyHex, string _)
     {
+        if (recoverySigs?.Signatures.Any() ?? false) //multiple relays for the same message
```

Comment:
I wonder if we should compare sigs (probably no need) we filter using pub key which is unique per user per project right?

Reply:
The sigs will be different for the same user and project as well